jacoco-maven-plugin/src/org/jacoco/maven/AbstractReportMojo.java

```diff
@@ -53,52 +36,67 @@
 	 *            default-value="UTF-8"
 	 */
 	String outputEncoding;
+
+	/**
+	 * Name of the root node HTML report pages.
+	 * 
+	 * @parameter default-value="${project.name}"
+	 * @since 0.7.7
+	 */
+	String title;
+
+	/**
+	 * Footer text used in HTML report pages.
+	 * 
+	 * @parameter
```

Comment:
I think would be nice to allow definition of values for `title` and `footer` when `report` mojo executed from command-line.